src/components/menu/menu.tsx

```diff
@@ -0,0 +1,229 @@
+import { Component, Element, h, Host, Listen, Method, Prop, State, Watch } from "@stencil/core";
+import { focusElement, focusElementInGroup, slotChangeGetAssignedElements } from "../../utils/dom";
+import {
+  componentLoaded,
+  LoadableComponent,
+  setComponentLoaded,
+  setUpLoadableComponent
+} from "../../utils/loadable";
+import { connectLocalized, disconnectLocalized, LocalizedComponent } from "../../utils/locale";
+import {
+  connectMessages,
+  disconnectMessages,
+  setUpMessages,
+  T9nComponent,
+  updateMessages
+} from "../../utils/t9n";
+import { MenuMessages } from "./assets/menu/t9n";
+
+type Layout = "horizontal" | "vertical";
+
+@Component({
+  tag: "calcite-menu",
+  styleUrl: "menu.scss",
+  shadow: true,
```

Comment:
should this component use delegatesFocus?

Reply:
nope, clicking on the non-focusable area in the menu shouldn't focus any element .